its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -1056,9 +1030,6 @@ void stop() {
     }
 
     @Test
-    // SonarQube should support opening security hotspots
-    @OnlyOnSonarQube(from = "8.6")
-    @Disabled
```

Comment:
I am not sure why it was disabled. I tried to enable it, and now it passes

Reply:
It was suddenly making the pipeline fail a few days ago, my guess was that it was because the rule has been deprecated and it wasn't loaded anymore

Reply:
Thanks for the heads up. It looks like it's working again